src/Command/ClearCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCacheBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCacheBundle\Command;
+
+use FOS\HttpCacheBundle\CacheManager;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * A command to trigger cache invalidation by path from the command line.
+ *
+ * @author Alexander Schranz <alexander@sulu.io>
+ */
+class ClearCommand extends BaseInvalidateCommand
+{
+    use PathSanityCheck;
+
+    protected static $defaultName = 'fos:httpcache:invalidate';
+
+    /**
+     * If no cache manager is specified explicitly, fos_http_cache.cache_manager
+     * is automatically loaded.
+     *
+     * @param CacheManager|null $cacheManager The cache manager to talk to
+     */
+    public function __construct(CacheManager $cacheManager = null)
+    {
+        parent::__construct($cacheManager);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName(static::$defaultName) // BC with 2.8
+            ->setDescription('Invalidate the whole http cache.')
+            ->setHelp(
+                <<<'EOF'
+The <info>%command.name%</info> command invalidates the whole cache in the configured caching proxies.
+
+Example:
+
+    <info>php %command.full_name%</info>
+EOF
+            )
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->getCacheManager()->clearCache();
```

Comment:
i suggest to do a check if the cache is clear capable here, then fallback to banning everything if the cache is ban capable, and finally report an explanation if neither is supported.